index.html

```diff
@@ -13,20 +13,20 @@
   </head>
   <body>
     <header>
-      <section id="session">
-        <a class="sess-link" href="#"><i class="sess-icon"><img class="face" src="images/icons/facebook-icon.png" alt="facebook icon"></i></a>
-        <a class="sess-link" href="#"><i class="sess-icon"><img class="twit" src="images/icons/twitter-icon.png" alt="twitter icon"></i></a>
-        <a class="sess-link" href="#"><p class="sess-icon">English</p></a>
-        <a class="sess-link" href="#"><p class="sess-icon">MyPage</p></a>
-        <a class="sess-link" href="#"><p class="sess-icon">Logout</p></a>
-      </section>
+      <ul id="session">
+        <li href="#"><a class="sess-link"><i><img class="face" src="./images/icons/facebook-icon.png" alt="facebook icon"></i></a></li>
+        <li href="#"><a class="sess-link"><i><img class="twit" src="./images/icons/twitter-icon.png" alt="twitter icon"></i></a></li>
+        <li href="#"><a class="sess-link">English</a></li>
+        <li href="#"><a class="sess-link">MyPage</a></li>
+        <li href="#"><a class="sess-link">Logout</a></li>
+      </ul>
       <label class="menu-btn">
-        <input type="checkbox">
+        <input id="checkbox" type="checkbox">
         <span class="burguer"></span>
         <nav>
-          <a href="#"><img class="nav-logo" src="images/content/ts-logo.png" alt="Tecate Supremo logo"></a>
+          <a href="#"><img class="nav-logo" src="./images/content/ts-logo.png" alt="Tecate Supremo logo"></a>
           <ul>
-            <li class="nav-item"><a href="pages/about.html" class="nav-link">About</a></li>
+            <li class="nav-item"><a href="./pages/about.html" class="nav-link">About</a></li>
             <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
             <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
             <li class="nav-item"><a href="#" class="nav-link">Sponsors</a></li>
```

Comment:
- [ OPTIONAL] Let make this look beautiful as  it is on the original design , on your perfomers , try to fix this so that your design look the same as the original one :wink:  
your design:
![Screenshot from 2023-02-10 23-27-34](https://user-images.githubusercontent.com/88894525/218203161-d457f482-7a30-405a-a8d8-3e85088657e3.png)

Figma Design
![figmabg](https://user-images.githubusercontent.com/88894525/218203194-daf9d148-4a62-4896-8d12-deab27eb1487.png)